op-node/rollup/derive/system_config.go

```diff
@@ -83,14 +83,19 @@ func ProcessSystemConfigUpdateLogEvent(destSysCfg *eth.SystemConfig, ev *types.L
 		if length, err := solabi.ReadUint64(reader); err != nil || length != 32 {
 			return NewCriticalError(errors.New("invalid length field"))
 		}
-		address, err := solabi.ReadAddress(reader)
+		version, address, err := solabi.ReadBatcherHashWithVersion(reader)
 		if err != nil {
-			return NewCriticalError(errors.New("could not read address"))
+			return NewCriticalError(errors.New("could not read batcher hash"))
 		}
-		if !solabi.EmptyReader(reader) {
-			return NewCriticalError(errors.New("too many bytes"))
+		if version > 1 {
+			return NewCriticalError(errors.New("unsupported batcher hash version"))
 		}
-		destSysCfg.BatcherAddr = address
+		// Update the batcher address if the version byte is 0 (version 1)
+		if version == 0 {
```

Comment:
Should we set the `BatcherAddr` to zero for version 1?

Reply:
According to what we [discussed](https://www.notion.so/oplabs/Meeting-eab6657803154f6b9c39b47445d60ac4?pvs=4#5f4ab8067b58450e895de8e0cd615b7d) last Friday, I think so.

Reply:
Maybe to a specific value for debugging...

Reply:
Ended up doing this https://github.com/EspressoSystems/op-leader-election/pull/23/commits/b6eb29a18050947a1932982ca5a77ca5058bf981